tools/install_builder_prerequisites.sh

```diff
@@ -31,12 +31,14 @@ function usage
 
 ASSUME_YES="false"
 SKIP_PATH_CHECK="false"
+OMIT_SUDO="false"
```

Comment:
This feels sketchy to me.  But I don't think it's a security issue (we were already running with `sudo` by default) and it's good that we don't have people run the whole script under `sudo` so I'm not sure what to do better.